### PR TITLE
Add deployment repo doc

### LINKED ISSUE #2381: Customer repository structure and how to consume the Azure TRE project
Sample deployment repository to be created.

AzureTRE repo docs to reference the repo with instructions on how to use.

### PR DESCRIPTION
**Pull request by LizaShak**
## What is being addressed

Add a documentation regarding AzureTRE deployment repository

### CODE REVIEW

File: docs/tre-admins/deployment-repo.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# TRE Deployment
+
+This document will cover how to extend AzureTRE with your custom images and deploy it.
+
+## AzureTRE deployment repo
+
+AzureTRE has an OSS deployment repository which you can find [here.](https://github.com/LizaShak/AzureTRE-Deployment)
```
Comment (marrobi) [inline docs/tre-admins/deployment-repo.md:7]: This URL needs updating?

Comment (LizaShak) [reply docs/tre-admins/deployment-repo.md:7]: ahh you are right. Fixed!

**Review by tamirkamara — APPROVED** [review]

**Pull request merged**